app/src/main/java/intellistart/interviewplanning/controllers/grpc/GrpcService.kt

```diff
@@ -0,0 +1,50 @@
+package intellistart.interviewplanning.controllers.grpc
+
+import intellistart.interviewplanning.controllers.dto.toProto
+import intellistart.interviewplanning.controllers.nats.slot.event.NatsEventSubscriber
+import intellistart.interviewplanning.grpc.slot_service.ReactorSlotServiceGrpc
+import intellistart.interviewplanning.model.slot.Slot
+import intellistart.interviewplanning.model.slot.SlotService
+import intellistart.interviewplanning.output.pubsub.slot.SlotUpdatedEvent
+import intellistart.interviewplanning.request.slot.stream_by_slot_id.StreamBySlotIdRequest
+import intellistart.interviewplanning.request.slot.stream_by_slot_id.StreamBySlotIdResponse
+import net.devh.boot.grpc.server.service.GrpcService
+import org.bson.types.ObjectId
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+
+@GrpcService
+class GrpcService(
```

Comment:
```suggestion
class SlotGrpcService(
```
you can have many grpc services